packages/common/src/store/gated-content/sagas.ts

```diff
@@ -647,56 +646,6 @@ function* updateSpecialAccessTracks(
   )
 }
 
-/**
- * 1. Get follow-gated tracks of unfollowed user
- * 2. Set those track statuses to 'LOCKED'
- * 3. Revoke access for those tracks
- */
-function* handleUnfollowUser(
```

Comment:
great job removing all this! fwiw im not opposed to keeping these sagas separate from the main follow flow to keep the hooks simpler for now, just a note for the future, since are already dispatching actions in the hook.